<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日期时间</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="btnGetTime">
        <property name="text">
         <string>读取当前日期时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>字符串显示</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="editTime"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnSetTime">
        <property name="text">
         <string>设置时间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="editDate"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnSetDate">
        <property name="text">
         <string>设置日期</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>日期时间</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="editDateTime"/>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnSetDateTime">
        <property name="text">
         <string>设置日期时间</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>日历选择</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>选择的日期:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editCalendar"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnQuit">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>定时器</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labelElapsTime">
        <property name="text">
         <string>流逝时间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>定时周期</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QSpinBox" name="intvEdit">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QPushButton" name="btnSetIntv">
        <property name="text">
         <string>设置周期</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLCDNumber" name="LCDHour"/>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QLCDNumber" name="LCDMin"/>
      </item>
      <item row="2" column="5">
       <widget class="QLCDNumber" name="LCDSec"/>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
